CMakeLists.txt

```diff
@@ -8,7 +8,12 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
-
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
This is a global configuration: remove it to allow the user to pass `-DCMAKE_BUILD_TYPE=Debug` when needed.